lib/model/list_model.dart

```diff
@@ -25,39 +29,41 @@ class ListModel {
   Iterable<Item> get scheduledItems =>
       itemsView().where((item) => item.isScheduled);
 
+  @ignore
+  Stream<ListModelEvent> get eventStream => _eventStreamController.stream;
```

Comment:
In the future, we might want to include the item that was modified in the stream. For now, just including the type of change is fine for our purposes.